src/components/FileRepo/FileRepo.js

```diff
@@ -53,27 +53,25 @@ const customTableTypes = {
   ),
 };
 
-const generateHeaderContent = (props, sqon) => {
-  props.disabled = false; //props.selectedTableRows == 0; @NOTE Keep functionality for next iteration of the ticket
-  props.sqon = sqon;
-  const cavaticaButton = (<Tooltip
+const TableHeaderContent = (props, sqon, disabled) => { return (
```

Comment:
I think you mean:
```
const TableHeaderContent = ({ sqon, disabled, ...props }) =>{
  return (...)
}
```
components receive props under one single object, not as a sequence of arguments

Reply:
can be done like that too, sure.